packages/rocketchat-ui-message/message/popup/messagePopup.coffee

```diff
@@ -108,6 +108,9 @@ Template.messagePopup.onCreated ->
 
 			template.enterValue()
 
+			if template.data.cls is 'search-results-list'
```

Comment:
I used the class because toolbar is the only template that will behave like that, but we can make a config for future use.